mgmt/rest/v2/config.go

```diff
@@ -159,3 +205,66 @@ func getPluginType(t string) (core.PluginType, error) {
 	}
 	return ityp, nil
 }
+
+// deletePluginConfigItemHelper builds different forms of request data into the way method deletePluginConfigItem accepts.
+// currently it accepts go-swagger client, swagger-ui and SanpCLI.
+func deletePluginConfigItemHelper(r *http.Request) []string {
+	buf, _ := ioutil.ReadAll(r.Body)
+	rdr2 := ioutil.NopCloser(bytes.NewBuffer(buf))
```

Comment:
do we need intermediate variable here? It couldn't be assigned to r.Body in line 237?